models/ellipsis_finance/bnb/ellipsis_finance_bnb_underlying_tokens.sql

```diff
@@ -0,0 +1,74 @@
+{{config(alias='underlying_tokens')}}
+
+WITH
+
+pool_underlying as (  
+    SELECT  
+        LOWER(pool) as pool, 
+        token_id,
+        LOWER(token_address) as token_address
+    FROM (
+    VALUES 
+    -- val3EPS 
+        ('0x5b5bD8913D766D005859CE002533D4838B0Ebbb5', '1', '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56'),
+        ('0x5b5bD8913D766D005859CE002533D4838B0Ebbb5', '2', '0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d'),
+        ('0x5b5bD8913D766D005859CE002533D4838B0Ebbb5', '3', '0x55d398326f99059fF775485246999027B3197955'), 
```

Comment:
Continue the above comment, [0x5b5bD8913D766D005859CE002533D4838B0Ebbb5](https://bscscan.com/token/0x5b5bd8913d766d005859ce002533d4838b0ebbb5) is a proof of stake, and a token at the same time, for a 3eps pool (BUSD/USDC/USDT). This particular pool would be a base pool by definition. 

The corresponding base pool is actually in the `FactoryPool_evt_BasePoolAdded` event table, which is:

contract_address | evt_tx_hash | evt_index | evt_block_time | evt_block_number | base_pool
-- | -- | -- | -- | -- | --
0xa5d748a3234a81120df7f23c9ea665587dc8d871 | 0x3ff0100a4e96d02b88bbc8f4d4457ead6df30a005ef04d596c79acb235624e04 | 279 | 2022-08-11 14:34 | 20350192 | 0x19ec9e3f7b21dd27598e7ad5aae7dc0db00a806d

(If you look into the `BEP20 Token Txns` in [BSCSCAN](https://bscscan.com/address/0x19ec9e3f7b21dd27598e7ad5aae7dc0db00a806d#tokentxns), you may have some ideas.)

Reply:
ah I see, so we'd still need to manually input the base pool tokens then, seeing as there are just two base pools, I think that it is fine. 

The base pool address is also different from the `token` (0x5b.....bbb5) that shows up in the metapool & plain pool deployed events. Hence why we need the manual tagging for that, which is currently the underlying tokens spell (nothing to change there imo).

The problem now is rewriting the pool_tokens spell which gets the tokens of the pool, that is currently pulled from the `call coins` source but I'd update that today once the factoryPool_v2 I submitted gets decoded on dune. 

The plainpool_deployed & the metapool_deployed events will allow us to get the coins of each pool.